### PR TITLE
ONE 11 cards

### PR DESCRIPTION
**Pull request by Simisays**
Should be good

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/churning_reservoir.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Churning Reservoir
+ManaCost:R
+Types:Artifact
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigPutCounter | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, put an oil counter on another target nontoken artifact or creature you control.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Artifact.nonToken+YouCtrl+Other,Creature.YouCtrl+Other | CounterType$ OIL| CounterNum$ 1
+A:AB$ Token | Cost$ 2 T | TokenScript$ r_1_1_phyrexian_goblin | CheckSVar$ CountCountersRemoved | CheckSVarCompare$ GE1 | SpellDescription$ Create a 1/1 red Phyrexian Goblin creature token. Activate only if an oil counter was removed from a permanent you controlled this turn or a permanent with an oil counter on it was put into a graveyard this turn.
+SVar:CountCountersRemoved:Count$Valid CountersRemovedThisTurn
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/churning_reservoir.txt:7]: Hornbeetle has this:
```
SVar:X:Count$CountersAddedThisTurn P1P1 You Creature.YouCtrl
```

similar is CountersRemovedThisTurn implemented but without "you"
```
SVar:CountCountersRemoved:CountersAddedThisTurn OIL Card.YouCtrl
```

but you also need to check for "Card with Oil Counter was send to graveyard"

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/churning_reservoir.txt:7]: Thanks for the quick reply, will try

File: forge-gui/res/cardsfolder/upcoming/ichor_synthesizer.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Ichor Synthesizer
+ManaCost:1 U
+Types:Creature Phyrexian Wizard
+PT:1/3
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | Execute$ TrigPutCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a noncreature spell, put an oil counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | CounterType$ OIL
+S:Mode$ Continuous | Affected$ Card.Self+counters_GE4_OIL | AddPower$ 2 | Description$ As long as CARDNAME has four or more oil counters on it, it gets +2/+0 and can't be blocked.
+S:Mode$ CantBlockBy | ValidAttacker$ Card.Self |
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/ichor_synthesizer.txt:8]: The CantBlockBy is incomplete

File: forge-gui/res/cardsfolder/upcoming/goliath_hatchery.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Name:Goliath Hatchery
+ManaCost:4 G G
+Types:Enchantment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create two 3/3 green Phyrexian Beast creature tokens with toxic 1. (Players dealt combat damage by them also get a poison counter.)
+SVar:TrigToken:DB$ Token | TokenAmount$ 2 | TokenScript$ g_3_3_phyrexian_beast_toxic
+T:Mode$ Phase | Phase$ Upkeep  | TriggerZones$ Battlefield | ValidPlayer$ You | Execute$ TrigChooseCard | CheckSVar$ X | SVarCompare$ GE3 | TriggerDescription$ Corrupted — At the beginning of your upkeep, if an opponent has three or more poison counters, choose a creature you control, then draw cards equal to its total toxic value.
+SVar:TrigChooseCard:DB$ ChooseCard | Defined$ You | Amount$ 1 | Choices$ Creature.YouCtrl | SubAbility$ DBDraw
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/goliath_hatchery.txt:7]: needs cleanup

File: forge-gui/res/cardsfolder/upcoming/predation_steward.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Predation Steward
+ManaCost:1 G
+Types:Creature Phyrexian Elf Warrior
+PT:2/2
+K:etbCounter:OIL:2
+A:AB$ Pump | Cost$ 2 G T SubCounter<1/OIL> | ValidTgts$ Creature | SorcerySpeed | NumAtt$ 2 | NumDef$ 2 | SpellDescription$ Target creature gets +2/+2 until end of turn. Activate only as a sorcery.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/predation_steward.txt:6]: "Speed$ True"

File: forge-gui/res/cardsfolder/upcoming/serum_core_chimera.txt [hunk 5]
```diff
@@ -0,0 +1,13 @@
+Name:Serum-Core Chimera
+ManaCost:2 U R
+Types:Creature Phyrexian Chimera
+PT:2/4
+K:Flying
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigCounter | TriggerDescription$ Whenever you cast a noncreature spell, put an oil counter on CARDNAME.
+SVar:TrigCounter:DB$ PutCounter | Defined$ Self | CounterType$ OIL | CounterNum$ 1
+A:AB$ Draw | Cost$ SubCounter<3/OIL> | SorcerySpeed$ True | SubAbility$ DBDiscard | SpellDescription$ Draw a card. Then you may discard a nonland card. When you discard a card this way, CARDNAME deals 3 damage to target creature or planeswalker. Activate only as a sorcery.
+SVar:DBDiscard:DB$ Discard | NumCards$ 1 | Mode$ TgtChoose | RememberDiscarded$ True | SubAbility$ DBImmediateTrig
+SVar:DBImmediateTrig:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Card | SubAbility$ TrigDamage | TriggerDescription$  When you discard a card this way, CARDNAME deals 3 damage to target creature or planeswalker.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/serum_core_chimera.txt:10]: Careful, that must use Execute

SubAbility should be Cleanup though

File: forge-gui/res/cardsfolder/upcoming/churning_reservoir.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/churning_reservoir.txt]: still incomplete like Hanmac said

File: forge-gui/res/cardsfolder/upcoming/churning_reservoir.txt [hunk 6]
```diff
@@ -0,0 +1,9 @@
+Name:Churning Reservoir
+ManaCost:R
+Types:Artifact
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigPutCounter | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, put an oil counter on another target nontoken artifact or creature you control.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Artifact.nonToken+YouCtrl+Other,Creature.YouCtrl+Other | CounterType$ OIL| CounterNum$ 1
+A:AB$ Token | Cost$ 2 T | TokenScript$ r_1_1_phyrexian_goblin | CheckSVar$ CountCountersRemoved | CheckSVarCompare$ GE1 | SpellDescription$ Create a 1/1 red Phyrexian Goblin creature token. Activate only if an oil counter was removed from a permanent you controlled this turn or a permanent with an oil counter on it was put into a graveyard this turn.
+SVar:CountCountersRemoved:Count$CountersRemovedThisTurn OIL Card.YouCtrl
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/churning_reservoir.txt:7]: Alright, here's a silly thing :D

_Skullbriar_ with oil counter -> cast _Suspend_ on him -> Use _Clockspinning_ to remove oil counter in Exile

Can you activate it?

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/churning_reservoir.txt]: Yes I can, is this how it's supposed to work ? :)

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/churning_reservoir.txt]: No, that's not allowed

you can probably add `+inRealZoneBattlefield` to the Valid for it

File: forge-gui/res/cardsfolder/upcoming/serum_core_chimera.txt [hunk 7]
```diff
@@ -0,0 +1,14 @@
+Name:Serum-Core Chimera
+ManaCost:2 U R
+Types:Creature Phyrexian Chimera
+PT:2/4
+K:Flying
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigCounter | TriggerDescription$ Whenever you cast a noncreature spell, put an oil counter on CARDNAME.
+SVar:TrigCounter:DB$ PutCounter | Defined$ Self | CounterType$ OIL | CounterNum$ 1
+A:AB$ Draw | Cost$ SubCounter<3/OIL> | SorcerySpeed$ True | SubAbility$ DBDiscard | SpellDescription$ Draw a card. Then you may discard a nonland card. When you discard a card this way, CARDNAME deals 3 damage to target creature or planeswalker. Activate only as a sorcery.
+SVar:DBDiscard:DB$ Discard | NumCards$ 1 | Mode$ TgtChoose | RememberDiscarded$ True | SubAbility$ DBImmediateTrig
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/serum_core_chimera.txt:9]: "discard nonland card"

File: forge-gui/res/cardsfolder/upcoming/serum_core_chimera.txt [hunk 8]
```diff
@@ -0,0 +1,14 @@
+Name:Serum-Core Chimera
+ManaCost:2 U R
+Types:Creature Phyrexian Chimera
+PT:2/4
+K:Flying
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigCounter | TriggerDescription$ Whenever you cast a noncreature spell, put an oil counter on CARDNAME.
+SVar:TrigCounter:DB$ PutCounter | Defined$ Self | CounterType$ OIL | CounterNum$ 1
+A:AB$ Draw | Cost$ SubCounter<3/OIL> | SorcerySpeed$ True | SubAbility$ DBDiscard | SpellDescription$ Draw a card. Then you may discard a nonland card. When you discard a card this way, CARDNAME deals 3 damage to target creature or planeswalker. Activate only as a sorcery.
+SVar:DBDiscard:DB$ Discard | NumCards$ 1 | DiscardValid$ Card.nonLand | DiscardValidDesc$ Select up to one nonland card in your hand to discard | Mode$ TgtChoose | RememberDiscarded$ True | SubAbility$ DBImmediateTrig
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/serum_core_chimera.txt:9]: are you sure that's optional...?

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/serum_core_chimera.txt:9]: `Then you may discard a nonland card.`

File: forge-gui/res/cardsfolder/upcoming/churning_reservoir.txt [hunk 9]
```diff
@@ -0,0 +1,10 @@
+Name:Churning Reservoir
+ManaCost:R
+Types:Artifact
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigPutCounter | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, put an oil counter on another target nontoken artifact or creature you control.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Artifact.nonToken+YouCtrl+Other,Creature.YouCtrl+Other | CounterType$ OIL| CounterNum$ 1
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/churning_reservoir.txt:5]: the nonToken should apply to both

**Review by tool4ever — APPROVED** [review]

**Pull request merged**